servers/SR430 Scalar Server.py

```diff
@@ -127,76 +130,108 @@ def clear_scan(self, c):
         dev = self.selectedDevice(c)
         yield dev.write('CLRS')
 
-    @setting(18, 'Start New Scan')
+    @setting(18, 'Start New Scan', returns='s')
```

Comment:
Add an input argument here which is the wait time, and remove the calculated wait time below.